libs/search-widget/src/widgets/search-widget/Widget.svelte

```diff
@@ -136,21 +141,22 @@
 <div class="nuclia-widget" {style} data-version="__NUCLIA_DEV_VERSION__">
   {#if ready}
     {#if type === 'input'}
-      <PopupSearch {placeholder}/>
+      <PopupSearch {placeholder} />
     {:else if type === 'form'}
-      <EmbeddedSearch {placeholder}/>
+      <EmbeddedSearch {placeholder} />
     {:else}
       {type} widget is not implemented yet
     {/if}
-    <Modal show={showModal}
-           on:close={closeModal}
-           closeButton={true}
-           --modal-width="var(--resource-modal-width)"
-           --modal-width-md="var(--resource-modal-width-md)"
-           --modal-height="var(--resource-modal-height)"
-           --modal-height-md="var(--resource-modal-height-md)"
+    <Modal
+      show={showModal}
+      on:close={closeModal}
+      closeButton={true}
+      --modal-width="var(--resource-modal-width)"
+      --modal-width-md="var(--resource-modal-width-md)"
+      --modal-height="var(--resource-modal-height)"
+      --modal-height-md="var(--resource-modal-height-md)"
```

Comment:
I really don't like this kind of reformatting. It seems like our IDE doesn't play using the same rules: seems like yours is forcing to have the attributes below the tag in svelte.